server/data/educationAndWorkPlanClient.ts

```diff
@@ -68,6 +69,13 @@ export default class EducationAndWorkPlanClient {
     })
   }
 
+  async completeGoal(prisonNumber: string, completeGoalRequest: CompleteGoalRequest, token: string): Promise<void> {
```

Comment:
Might be worth considering a unit test for this new method, but we can back fill it in a subsequent PR 👍